tests/test_types.py

```diff
@@ -202,3 +202,169 @@ class Query:
     field = type_def.get_field("color")
     assert field
     assert field.type is ColorType
+
+
+@override_settings(
+    STRAWBERRY_DJANGO=StrawberryDjangoSettings(
+        FIELD_DESCRIPTION_FROM_HELP_TEXT=True,
+        TYPE_DESCRIPTION_FROM_MODEL_DOCSTRING=True,
+    ),
+)
+def test_all_fields_works():
+    @strawberry_django.type(Fruit, fields="__all__")
+    class FruitType:
+        pass
+
+    type_def = get_object_definition(FruitType, strict=True)
+    assert type_def.description == "Fruit(id, name, color, sweetness)"
+
+    name_field = type_def.get_field("name")
+    assert name_field is not None
+    assert name_field.type == str
+
+    color_field = type_def.get_field("color")
+    assert color_field is not None
+
+    sweetness_field = type_def.get_field("sweetness")
+    assert sweetness_field is not None
+    assert sweetness_field.type == int
+    assert sweetness_field.description == Fruit._meta.get_field("sweetness").help_text
```

Comment:
Another way of testing this all would be to test the printed schema.

You can see in the `tests/test_queries.py` how `test_field_name` does that as an example.

Since those are tests I'm fine with not doing that, but I'll leave this comment here in case you want to do that :)

Reply:
Thank you for the suggestion, I like the printed schema method better 👍